site/index.html

```diff
@@ -76,6 +76,62 @@ <h2>Try it!</h2>
 
 <br>
 
+<div class="features">
+   <div class=" container ">
+      <h2>Why Haskell?</h2>
+      <div class=" row ">
+         <div class=" span6 col-md-6">
+            <h3>A new paradigm</h3>
+            <p>Express your ideas clearly and learn a new way of thinking about programming. Based on lambda calculus Haskell is a
+            purely functional programming language that features referential transparency, immutability and lazy evaluation.
+            Concepts that will blow your mind
+               &mdash; relearn programming while having an absolute blast.
+            </p>
+         </div>
+         <div class=" span6 col-md-6">
+            <h3>Composition and predictability</h3>
+            <p>Reason about large pieces of code and compose them easily. There is no global state
+            or mutable variables obscuring the meaning of your program. The strong type system makes sure there are no surprises
+               &mdash; never again will you have to guess what your program does at execution time.
+            </p>
+         </div>
+      </div>
+      <div class=" row ">
+         <div class=" span6 col-md-6">
+           <h3>Declarative</h3>
+           <p>Write your programs declaratively by utilizing the power of pure functions and algebraic data types.
+              In Haskell we don't write how a program should be executed, we just describe its logic
+                 &mdash; never again be forced to think about evaluation order or execution details.
+           </p>
+         </div>
+         <div class=" span6 col-md-6">
+            <h3>Performance</h3>
+            <p>Squeeze out the last ticks of your multi-core processors, thanks to best-in-class support for async,
+            concurrent and parallel programming... made possible via garbage collection and green threads.
+            Use advanced streaming libraries for ultra efficient data processing.
+            </p>
+         </div>
+      </div>
+      <div class=" row ">
+         <div class=" span6 col-md-6">
+            <h3>Abstraction</h3>
+            <p>Build powerful abstractions that are not possible in other languages. Only your imagination is the limit, due to
+            polymorphism, type classes and more advanced typesystem features.
+            Haskell has its roots in programming language research and will always be at the forefront of expressivity.
+            </p>
+         </div>
+         <div class=" span6 col-md-6">
+            <h3>Excellent tooling</h3>
+            <p>A tooling story that's truly amazing: spawn your toolchain with GHCup, build your project with cabal, get editor integration with haskell-language-server
```

Comment:
Feels a little hyperbolic. How about a more understated: "Simple, effective and modern tooling: spwan ..."

Reply:
"Simple" is a rather controversial property when it comes to tooling and it's unclear to me what "effective" would mean.

I think having an amazing tooling story is our goal. Yes, we're overstating a bit.

Again: the hyped up language is on purpose. Your proposed wording makes it a bit more dry, in my opinion. Those are different styles.

Reply:
That's fair. But while I think Haskell's tooling is very good (especially compared to 10 years ago when I started using Haskell), is it really a core feature that should be used as an argument for adoption? It seems to me that the obvious way for the tooling to shine is that it's now easy for people to spend 10 minutes trying out Haskell, and have an straightforward time getting set up. 

I'm not opposed to less dry language, stylistically speaking, but this seems like an odd target for hyperbole. Should tooling even aim to be amazing (rather than just so straightforward that you never have to think about it)? The things I'd want to be overstated about are how Haskell is pure, makes refactoring complex code easy, etc.